src/components/landing/LandingContent.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { TextData } from '../../api/texts';
+
+const tokenImage = require('../../graphics/landing_token.svg');
+const mobileImage = require('../../graphics/landing_phone.png');
+const labelValues = [];
+
+class LandingContent extends React.Component {
```

Comment:
You dont need a component class for this, this does not have any state.
You just need a higher order function. Meaning that the funtion olny returns the JSX, but you build around the function the proper methods needed.
This is too much for just a simple view component

Reply:
Sure. Will do the necessary changes.